codeforamerica/vita-min
# Nj 151 nj2450

## Link to pivotal/JIRA issue
https://github.com/newjersey/affordability-pm/issues/151

## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

## What was done?
- Add NJ 2450 xml and PDF
- Add fields to state file w2

## How to test?
- Select a profile with multiple w2s
- Alter box 14 UI/WF/SWF and FLI values ($90 for each in each W2 should make it so taxpayer only gets excess FLI back; using $100 for UI/WF/SWF should ensure you see both)
- Finish the process and submit
- Check out xml and pdf submissions

## Screenshots (for visual changes)
![Screenshot 2024-11-18 at 17-44-21 ](https://github.com/user-attachments/assets/50e5d867-c8b0-402d-ab49-8264027caf2f)
![Screenshot 2024-11-18 at 17-18-23 submission pdf](https://github.com/user-attachments/assets/10f5a563-60ee-4f7a-9bae-ed14c894e4e3)
![Screenshot 2024-11-18 at 17-18-39 submission pdf](https://github.com/user-attachments/assets/ac5bc844-4c35-4242-8a3a-0b94c50c0f62)
